diff -p -up /etc/vigilo/vigiconf/conf.d/general/apps.py.orig /etc/vigilo/vigiconf/conf.d/general/apps.py
--- /etc/vigilo/vigiconf/conf.d/general/apps.py.orig	2010-02-16 15:58:39.000000000 +0100
+++ /etc/vigilo/vigiconf/conf.d/general/apps.py	2010-02-16 15:58:54.000000000 +0100
@@ -47,7 +47,7 @@
 appsByAppGroups = {
     'collect'           : ['nagios', 'collector', 'perfdata'],
     'metrology'         : ['connector-metro', 'rrdgraph'],
-    'trap'              : ['corrtrap'],
+    'trap'              : [],
     'correlation'       : ['nagios-hls'],
 }
 
 
