diff -Nru /etc/nagios.orig/nagios.cfg /etc/nagios/nagios.cfg
--- /etc/nagios.orig/nagios.cfg	2009-05-13 11:28:49.000000000 +0200
+++ /etc/nagios/nagios.cfg	2010-07-05 15:27:43.000000000 +0200
@@ -46,16 +46,22 @@
 #cfg_file=/etc/nagios/objects/printer.cfg
 
 # load configuration files on the fly
-cfg_dir=/etc/nagios/conf.d
+#cfg_dir=/etc/nagios/conf.d
 
 # You can also tell Nagios to process all config files (with a .cfg
 # extension) in a particular directory by using the cfg_dir
 # directive as shown below:
 
-cfg_dir=/etc/nagios/servers
-cfg_dir=/etc/nagios/printers
-cfg_dir=/etc/nagios/switches
-cfg_dir=/etc/nagios/routers
+#cfg_dir=/etc/nagios/servers
+#cfg_dir=/etc/nagios/printers
+#cfg_dir=/etc/nagios/switches
+#cfg_dir=/etc/nagios/routers
+
+# Vigilo specific file
+cfg_file=/etc/nagios/vigilo.cfg
+
+# Vigiconf-generated conf file
+cfg_dir=/etc/vigilo/vigiconf/prod/nagios
 
 
 # OBJECT CACHE FILE
@@ -338,7 +344,7 @@
 # this value to 0.  If passive checks should be logged, set
 # this value to 1.
 
-log_passive_checks=1
+log_passive_checks=0
 
 
 
@@ -379,7 +385,7 @@
 # program start time that an initial check of all services should
 # be completed.  Default is 30 minutes.
 
-max_service_check_spread=30
+max_service_check_spread=5
 
 
 
@@ -440,7 +446,7 @@
 # This is the frequency (in seconds!) that Nagios will process
 # the results of host and service checks.
 
-check_result_reaper_frequency=10
+check_result_reaper_frequency=3
 
 
 
@@ -607,8 +613,8 @@
 # ocsp command, and performance data commands.  All values are in
 # seconds.
 
-service_check_timeout=60
-host_check_timeout=30
+service_check_timeout=30
+host_check_timeout=10
 event_handler_timeout=30
 notification_timeout=30
 ocsp_timeout=5
@@ -752,7 +758,7 @@
 # on what aggressive host check is or check out the source code in
 # base/checks.c
 
-use_aggressive_host_checking=0
+use_aggressive_host_checking=1
 
 
 
@@ -829,7 +835,7 @@
 # performance data.
 # Values: 1 = process performance data, 0 = do not process performance data
 
-process_performance_data=0
+process_performance_data=1
 
 
 
@@ -841,8 +847,8 @@
 # define in your host configuration file.  Read the HTML docs for
 # more information on performance data.
 
-#host_perfdata_command=process-host-perfdata
-#service_perfdata_command=process-service-perfdata
+host_perfdata_command=process-host-perfdata
+service_perfdata_command=process-service-perfdata
 
 
 
@@ -1013,7 +1019,7 @@
 # check the "freshness" of service check results.  If you have
 # disabled service freshness checking, this option has no effect.
 
-service_freshness_check_interval=60
+service_freshness_check_interval=10
 
 
 
@@ -1024,7 +1030,7 @@
 # manner.
 # Values: 1 = enabled freshness checking, 0 = disable freshness checking
 
-check_host_freshness=0
+check_host_freshness=1
 
 
 
@@ -1033,7 +1039,7 @@
 # check the "freshness" of host check results.  If you have
 # disabled host freshness checking, this option has no effect.
 
-host_freshness_check_interval=60
+host_freshness_check_interval=10
 
 
 
@@ -1060,7 +1066,7 @@
 # Values: 1 = enable flap detection
 #         0 = disable flap detection (default)
 
-enable_flap_detection=1
+enable_flap_detection=0
 
 
 
@@ -1085,7 +1091,7 @@
 #	strict-iso8601	(YYYY-MM-DDTHH:MM:SS)
 #
 
-date_format=us
+date_format=euro
 
 
 
@@ -1225,7 +1231,7 @@
 # Values: 1 - Enabled tweaks
 #         0 - Disable tweaks (default)
 
-use_large_installation_tweaks=0
+use_large_installation_tweaks=1
 
 
 
