--- nagios.cfg.orig	2017-03-01 17:57:23.517000000 +0100
+++ nagios.cfg	2017-03-01 18:09:49.515000000 +0100
@@ -26,13 +26,13 @@
 # if you wish (as shown below), or keep them all in a single config file.
 
 # You can specify individual object config files as shown below:
-cfg_file=/etc/nagios/objects/commands.cfg
-cfg_file=/etc/nagios/objects/contacts.cfg
-cfg_file=/etc/nagios/objects/timeperiods.cfg
-cfg_file=/etc/nagios/objects/templates.cfg
+#cfg_file=/etc/nagios/objects/commands.cfg
+#cfg_file=/etc/nagios/objects/contacts.cfg
+#cfg_file=/etc/nagios/objects/timeperiods.cfg
+#cfg_file=/etc/nagios/objects/templates.cfg
 
 # Definitions for monitoring the local (Linux) host
-cfg_file=/etc/nagios/objects/localhost.cfg
+#cfg_file=/etc/nagios/objects/localhost.cfg
 
 # Definitions for monitoring a Windows machine
 #cfg_file=/etc/nagios/objects/windows.cfg
@@ -55,6 +55,14 @@
 
 cfg_dir=/etc/nagios/conf.d
 
+# Command definitions for Nagios plugins
+cfg_dir=/etc/nagios/plugins.d/
+
+# Vigilo specific files
+cfg_dir=/etc/nagios/vigilo.d/
+
+# Vigiconf-generated conf files
+cfg_dir=/etc/vigilo/vigiconf/prod/nagios
 
 
 
@@ -104,7 +112,7 @@
 # The contents of the status file are deleted every time Nagios
 #  restarts.
 
-status_file=/var/spool/nagios/status.dat
+status_file=/dev/shm/nagios-status.dat
 
 
 
@@ -178,16 +186,14 @@
 # is created, used, and deleted throughout the time that Nagios is
 # running.
 
-temp_file=/var/spool/nagios/nagios.tmp
-
+temp_file=/dev/shm/nagios.tmp
 
 
 # TEMP PATH
 # This is path where Nagios can create temp files for service and
 # host check results, etc.
 
-temp_path=/tmp
-
+temp_path=/dev/shm
 
 
 # EVENT BROKER OPTIONS
@@ -317,7 +323,7 @@
 # checks - see the option below for controlling whether or not
 # passive checks are logged.
 
-log_external_commands=1
+log_external_commands=0
 
 
 
@@ -326,7 +332,7 @@
 # this value to 0.  If passive checks should be logged, set
 # this value to 1.
 
-log_passive_checks=1
+log_passive_checks=0
 
 
 
@@ -367,7 +373,7 @@
 # program start time that an initial check of all services should
 # be completed.  Default is 30 minutes.
 
-max_service_check_spread=30
+max_service_check_spread=5
 
 
 
@@ -428,7 +434,7 @@
 # This is the frequency (in seconds!) that Nagios will process
 # the results of host and service checks.
 
-check_result_reaper_frequency=10
+check_result_reaper_frequency=3
 
 
 
@@ -587,8 +593,8 @@
 # ocsp command, and performance data commands.  All values are in
 # seconds.
 
-service_check_timeout=60
-host_check_timeout=30
+service_check_timeout=30
+host_check_timeout=10
 event_handler_timeout=30
 notification_timeout=30
 ocsp_timeout=5
@@ -709,7 +715,7 @@
 # by Nagios Enterprises from the update check is processed in accordance 
 # with our privacy policy - see https://api.nagios.org for details.
 
-check_for_updates=1
+check_for_updates=0
 
 
 
@@ -809,7 +815,7 @@
 # performance data.
 # Values: 1 = process performance data, 0 = do not process performance data
 
-process_performance_data=0
+process_performance_data=1
 
 
 
@@ -821,8 +827,8 @@
 # define in your host configuration file.  Read the HTML docs for
 # more information on performance data.
 
-#host_perfdata_command=process-host-perfdata
-#service_perfdata_command=process-service-perfdata
+host_perfdata_command=process-host-perfdata
+service_perfdata_command=process-service-perfdata
 
 
 
@@ -902,7 +908,7 @@
 # implementing distributed monitoring.
 # Values: 1 = obsess over services, 0 = do not obsess (default)
 
-obsess_over_services=0
+obsess_over_services=1
 
 
 
@@ -914,7 +920,7 @@
 # define in your host configuration file. Read the HTML docs for
 # more information on implementing distributed monitoring.
 
-#ocsp_command=somecommand
+ocsp_command=ocsp-bus
 
 
 
@@ -926,7 +932,7 @@
 # implementing distributed monitoring.
 # Values: 1 = obsess over hosts, 0 = do not obsess (default)
 
-obsess_over_hosts=0
+obsess_over_hosts=1
 
 
 
@@ -938,7 +944,7 @@
 # define in your host configuration file. Read the HTML docs for
 # more information on implementing distributed monitoring.
 
-#ochp_command=somecommand
+ochp_command=ochp-bus
 
 
 
@@ -1005,7 +1011,7 @@
 # check the "freshness" of service check results.  If you have
 # disabled service freshness checking, this option has no effect.
 
-service_freshness_check_interval=60
+service_freshness_check_interval=10
 
 
 
@@ -1032,7 +1038,7 @@
 # manner.
 # Values: 1 = enabled freshness checking, 0 = disable freshness checking
 
-check_host_freshness=0
+check_host_freshness=1
 
 
 
@@ -1041,7 +1047,7 @@
 # check the "freshness" of host check results.  If you have
 # disabled host freshness checking, this option has no effect.
 
-host_freshness_check_interval=60
+host_freshness_check_interval=10
 
 
 
@@ -1068,7 +1074,7 @@
 # Values: 1 = enable flap detection
 #         0 = disable flap detection (default)
 
-enable_flap_detection=1
+enable_flap_detection=0
 
 
 
@@ -1093,7 +1099,7 @@
 #	strict-iso8601	(YYYY-MM-DDTHH:MM:SS)
 #
 
-date_format=us
+date_format=euro
 
 
 
@@ -1122,8 +1128,7 @@
 # be used in host names, service descriptions, or names of other
 # object types.
 
-illegal_object_name_chars=`~!$%^&*|'"<>?,()=
-
+illegal_object_name_chars=`~!$%^&*|"<>?,()=
 
 
 # ILLEGAL MACRO OUTPUT CHARACTERS
@@ -1204,7 +1209,7 @@
 # Values: 1 - Enabled tweaks
 #         0 - Disable tweaks (default)
 
-use_large_installation_tweaks=0
+use_large_installation_tweaks=1
 
 
 
