--- /etc/ejabberd/ejabberd.cfg.orig	2011-06-18 21:43:24.000000000 +0200
+++ /etc/ejabberd/ejabberd.cfg	2012-01-30 14:44:12.816274321 +0100
@@ -343,6 +343,7 @@
 %% The "fast" shaper limits traffic speed to 50000 B/s
 %%
 {shaper, fast, {maxrate, 50000}}.
+{shaper, veryfast, {maxrate, 100000000}}. % 100 MB/s
 
 %%
 %% This option specifies the maximum number of elements in the queue
@@ -408,7 +409,7 @@
 
 %% For C2S connections, all users except admins use the "normal" shaper
 {access, c2s_shaper, [{none, admin},
-		      {normal, all}]}.
+		      {veryfast, all}]}.
 
 %% All S2S connections use the "fast" shaper
 {access, s2s_shaper, [{fast, all}]}.
@@ -519,7 +520,14 @@
   %%{mod_pres_counter,[{count, 5}, {interval, 60}]},
   {mod_privacy,  []},
   {mod_private,  []},
-  %%{mod_proxy65,[]},
+  {mod_admin_extra,  []},
+  {mod_proxy65,  [
+		  {access, all},
+		  {host, "proxy65.localhost"},
+		  {ip, {0,0,0,0}},
+		  {name, "SOCKS5 BytesStreams"},
+		  {port , 7777}
+		 ]},
   {mod_pubsub,   [
 		  {access_createnode, pubsub_createnode},
 		  {ignore_pep_from_offline, true}, % reduces resource comsumption, but XEP incompliant
